memory/src/main/scala/filodb.memory/format/vectors/LongBinaryVector.scala

```diff
@@ -195,7 +195,7 @@ object LongVectorDataReader64 extends LongVectorDataReader {
     require(start >= 0 && end < length(vector), s"($start, $end) is out of bounds, length=${length(vector)}")
     var addr = vector + OffsetData + start * 8
     val untilAddr = vector + OffsetData + end * 8 + 8   // one past the end
-    var sum: Double = 0d
+    var sum: Double = Double.NaN
```

Comment:
Are you sure this is right?  There is no check for NaN when it adds the actual values.  I think this will lead to the sum always being NaN, which is not right.

Reply:
Sorry I missed this